packages/plugin-flex/src/commands/flex/plugins/start.ts

```diff
@@ -1,8 +1,10 @@
+/* eslint-disable complexity */
```

Comment:
remove this - these are just warnings and they are legit warnings -  we need to eventually fix them